types/highcharts/index.d.ts

```diff
@@ -5767,8 +5767,10 @@ declare namespace Highcharts {
          *            name: 'Point1',
          *            color: '#FF00FF'
          *        }]
+         *
+         * 4. An array of arrays with three values for ranges. In this case the values correspond x, yMin and yMax.
```

Comment:
I think this comment is intended to cover both new types, but it seems like it might be more consistent to explain them separately.

Reply:
Ok. I just assumed that since in part 2) the [string, number] was not mentioned, so I assumed that was implicit.
I forgot to provide an example in the comment. I just noticed I forgot to provide an example in the comment. Give me a sec, maybe I can fix both.